### PR TITLE
Removed logset from integrations

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Removed logset from integration sources

### Motivation

Do not enable users to override the logset value from integration files

### CODE REVIEW

File: pkg/logs/processor/processor.go [hunk 1]
```diff
@@ -18,27 +18,20 @@ import (
 // A Processor updates messages from an inputChan and pushes
 // in an outputChan
 type Processor struct {
-	inputChan    chan message.Message
-	outputChan   chan message.Message
-	apikey       string
-	logset       string
-	apikeyString []byte
```
Comment (tmichelet) [inline pkg/logs/processor/processor.go]: I would keep the `[]byte` version of `api_key/logset` to avoid computing it for each message

**Review by tmichelet — CHANGES_REQUESTED** [review]
One small thing to change to keep performance levels, otherwise looks good

File: pkg/logs/processor/processor.go (hunk 1 above)
Comment (ajacquemot) [reply pkg/logs/processor/processor.go]: done

File: pkg/logs/processor/processor.go [hunk 2]
```diff
@@ -52,9 +45,9 @@ func (p *Processor) run() {
 	for msg := range p.inputChan {
 		shouldProcess, redactedMessage := p.applyRedactingRules(msg)
 		if shouldProcess {
+			apiKey := p.apiKey
 			extraContent := p.computeExtraContent(msg)
-			apikeyString := p.computeAPIKeyString(msg)
-			payload := p.buildPayload(apikeyString, redactedMessage, extraContent)
+			payload := p.buildPayload(apiKey, redactedMessage, extraContent)
```
Comment (tmichelet) [inline pkg/logs/processor/processor.go]: why not using `p.apiKey` directly here?

**Review by tmichelet — APPROVED** [review]
minor comment, GTG

**Pull request merged**